### PR TITLE
- add support for Visual Studio 2019 (16)

### LINKED ISSUE #116: Add support for Visual Studio 2019 (currently in preview)
Visual Studio 2019 (Preview) comes with improved CMake support. Unfortunately, cmake-conan doesn't recognize it's compiler (details in conan-io/conan#4146). Added support for Visual Studio 2019 (Preview) would be much appreciated.

### PR DESCRIPTION
**Pull request by SSE4**
closes #116

### CODE REVIEW

File: conan.cmake [hunk 1]
```diff
@@ -318,15 +320,15 @@ function(conan_cmake_install)
         else()
             set(CONAN_BUILD_POLICY ${CONAN_BUILD_POLICY} --build=${ARG})
         endif()
-    endforeach()
+    endforeach()   
     if(ARGUMENTS_CONAN_COMMAND)
        set(conan_command ${ARGUMENTS_CONAN_COMMAND})
     else()
-      set(conan_command conan)
+      set(conan_command "C:\\bincrafters\\conan\\conan.bat")
```
Comment (mikedld) [inline conan.cmake]: This doesn't look right to me.

File: conan.cmake [hunk 2]
```diff
@@ -318,15 +320,15 @@ function(conan_cmake_install)
         else()
             set(CONAN_BUILD_POLICY ${CONAN_BUILD_POLICY} --build=${ARG})
         endif()
-    endforeach()
+    endforeach()
```
Comment (mikedld) [inline conan.cmake]: Please clean up the PR by trimming trailing whitespace (here and down below).

File: conan.cmake (hunk 1 above)
Comment (SSE4) [reply conan.cmake]: yeah, my fault, I didn't want to commit this, removed

File: conan.cmake (hunk 2 above)
Comment (SSE4) [reply conan.cmake]: yeah, my fault, I didn't want to commit this, removed

**Comment by heilkn** [comment]
@memsharded Could you please take a look, if this can be merged?

**Comment by JimmySnails** [comment]
What's the status on this? I could really need Studio 2019 support for my project :)

**Comment by jmarrec** [comment]
My two cents: You should try with this PR change (it's simple). You might be blocked by some recipes that aren't yet supporting msvc 2019 regardless of conan-cmake though.

**Comment by memsharded** [comment]
Sorry all for the long delays. Will gather open issues and PRs and do a release.
Thanks for the contributions!

**Pull request merged**